src/main.js

```diff
@@ -89,9 +89,12 @@ export default function start() {
   app.register(EnhancerToken, ReduxActionEmitterEnhancer);
 
   if (__NODE__) {
-    app.register(GetInitialStateToken, async () => {
-      return {};
-    });
+    app.register(
+      GetInitialStateToken,
+      async (): Object => {
```

Comment:
Hm, would this be more correct as `Promise<Object>` given that it's an `async` function?  I don't think Flow adds the implicit

Reply:
I suppose that makes sense! Updated, thanks.